### PR TITLE
Allow to disable particular items in ObjectPopup

### PR DESCRIPTION
**Pull request by denis-bunakalya**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

**Comment by haiodo** [comment]
@denis-bunakalya I do not like this idea at all.

**Comment by denis-bunakalya** [comment]
> @denis-bunakalya I do not like this idea at all.

Any reason? We need it to show all the selected items, but disable editing for some of them. If we would show only editable items, user sometimes won't see other previously selected items, which are not editable for him

**Comment by haiodo** [comment]
So you need to disallow deselection of some of items? 
In this case you need name it differently, since it is confusing.

And potentially it should be shown differently in control.

**Comment by denis-bunakalya** [comment]
> So you need to disallow deselection of some of items? In this case you need name it differently, since it is confusing.
> 
> And potentially it should be shown differently in control.

Done

File: packages/presentation/src/components/ObjectPopup.svelte [hunk 1]
```diff
@@ -58,6 +58,7 @@
 
   export let create: ObjectCreate | undefined = undefined
   export let readonly = false
+  export let forbiddenDeselectItemIds: Set<Ref<Doc>> = new Set()
```
Comment (haiodo) [inline packages/presentation/src/components/ObjectPopup.svelte]:
```suggestion
  export let disallowDeselect: Ref<Doc>[]| undefined
```

Comment (denis-bunakalya) [reply packages/presentation/src/components/ObjectPopup.svelte]: Done

**Review by haiodo — APPROVED** [review]

**Pull request merged**